server-script.py

```diff
@@ -0,0 +1,121 @@
+import threading
+from DB import dbclient
+from server import serverclient
+import time
+import logging
+import sys
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+server = serverclient.Server()
+clients = {}
+clients_lock = threading.Lock()
```

Comment:
you are initializing instances outside class and class method uses them and not self params